apollo-runtime/src/main/java/com/apollographql/apollo/ApolloClient.java

```diff
@@ -339,6 +395,24 @@ Response cachedHttpResponse(String cacheKey) throws IOException {
     Builder() {
     }
 
+    Builder(ApolloClient apolloClient) {
```

Comment:
```suggestion
    private Builder(@NotNull ApolloClient apolloClient) {
```

* private because there shouldn't be a reason to call it instead of `newBuilder()`, right ?
* @NotNull because the IDE suggested it 🙃